package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "root",
+  "private": true,
+  "workspaces": {
+    "packages": [
+      "packages/*"
+    ],
+    "nohoist": [
+      "**/@vue",
+      "**/@vue/**",
+      "**/vue-cli-plugin-vuetify",
+      "**/vue-cli-plugin-vuetify/**",
+      "**/vuetify-loader",
+      "**/vuetify-loader/**",
+      "**/vue-jest",
+      "**/vue-jest/**",
+      "**/vue-loading-overlay",
+      "**/vue-loading-overlay**",
+      "**/@mdi",
+      "**/@mdi/**",
+      "**/material-design-icons-iconfont",
+      "**/material-design-icons-iconfont/**",
+      "**/@sap-devx",
+      "**/@sap-devx/**"
+    ]
+  },
+  "scripts": {
+    "release:version": "lerna version",
+    "release:publish": "lerna publish from-git --yes",
+    "ci": "npm-run-all coverage:clean format:validate lint:validate legal:* ci:subpackages coverage:merge",
+    "ci:subpackages": "lerna run ci",
+    "compile": "lerna run clean && tsc --build",
+    "compile:watch": "lerna run clean && tsc --build --watch",
+    "format:fix": "prettier --write \"**/*.@(js|ts|json|md|vue)\" --ignore-path=.gitignore",
+    "format:validate": "prettier --check \"**/*.@(js|ts|json|md|vue)\" --ignore-path=.gitignore",
+    "lint:fix": "eslint . --ext=js,ts,vue --fix --max-warnings=0 --ignore-path=.gitignore",
+    "lint:validate": "eslint . --ext=js,ts,vue --max-warnings=0 --ignore-path=.gitignore",
+    "coverage:clean": "rimraf ./coverage",
+    "coverage:merge": "lcov-result-merger \"./coverage/lcov*.info\" \"./coverage/lcov.info\"",
+    "legal:delete": "lerna exec \"shx rm -rf .reuse LICENSES\" || true",
+    "legal:copy": "lerna exec \"shx cp -r ../../.reuse .reuse && shx cp -r ../../LICENSES LICENSES\""
+  },
+  "dependencies": {
+    "lodash": "4.17.21"
+  },
+  "devDependencies": {
+    "@types/chai": "4.2.14",
+    "@types/mocha": "8.2.0",
+    "@types/rimraf": "3.0.0",
+    "chai": "4.2.0",
+    "eslint": "7.18.0",
+    "@typescript-eslint/parser": "4.14.0",
+    "@typescript-eslint/eslint-plugin": "4.15.0",
+    "eslint-plugin-vue": "^7.3.0",
+    "eslint-config-prettier": "7.2.0",
+    "eslint-plugin-eslint-comments": "3.2.0",
+    "husky": "4.3.6",
+    "lerna": "3.22.1",
+    "lint-staged": "10.5.3",
+    "mocha": "8.2.1",
+    "npm-run-all": "4.1.5",
+    "nyc": "15.1.0",
+    "prettier": "2.2.1",
+    "typescript": "4.1.3",
+    "cz-conventional-changelog": "3.3.0",
+    "@commitlint/cli": "11.0.0",
+    "@commitlint/config-conventional": "11.0.0",
+    "rimraf": "3.0.2",
+    "coveralls": "2.11.16",
+    "shx": "0.3.3",
+    "patch-package": "^6.2.2",
```

Comment:
i do not htink you use patch-package.

Reply:
You are right, but when I removed it yesterday vsix stopped working. I'll try to understand why.